src/main/java/com/develhope/spring/users/components/UserMapper.java

```diff
@@ -0,0 +1,25 @@
+package com.develhope.spring.users.components;
+
+import com.develhope.spring.users.dtos.UserCreatorDTO;
+import com.develhope.spring.users.dtos.UserSavedDTO;
+import com.develhope.spring.users.models.User;
+import org.springframework.stereotype.Component;
+
+@Component
+public class UserMapper {
+
+    public UserSavedDTO fromEntityToUserSavedDTO(User user) {
```

Comment:
CI standardiziamo sul chiamare questi metodi con to*DTODiDestinazione*From(Class<T> class)